FLEDGE.md

```diff
@@ -1278,19 +1278,25 @@ The following table summarizes which APIs get original and which get converted b
 
 #### 5.4 Reporting IDs
 
-Protected Audience provides several interest group fields that can be used to report details about a bid.  Which of `selectableBuyerAndSellerReportingId`, `buyerAndSellerReportingId`, `buyerReportingId`, or the interest group name gets passed to `reportWin()` and `reportResult()` is determined by the browser with the following logic:
-
-If `selectableBuyerAndSellerReportingId` defined in interest group:<br>
-&nbsp;&nbsp;If `selectableBuyerAndSellerReportingId` in bid:<br>
-&nbsp;&nbsp;&nbsp;&nbsp;Then `selectableBuyerAndSellerReportingId`, `buyerAndSellerReportingId` (if present in interest group), and `buyerReportingId` (if present in interest group) will all be available to reporting.<br>
-&nbsp;&nbsp;Otherwise (i.e. when If `selectableBuyerAndSellerReportingId` not in bid):<br>
-&nbsp;&nbsp;&nbsp;&nbsp;No reporting IDs. If you want reporting IDs, consider including and selecting an empty `selectableBuyerAndSellerReportingId`.<br>
-Otherwise (i.e. when selectableBaRSI not defined in IG):<br>
-&nbsp;&nbsp;If `buyerAndSellerReportingId` defined in interest group: `buyerAndSellerReportingId` available to reporting.<br>
-&nbsp;&nbsp;Otherwise, if `buyerReportingId` defined in interest group: `buyerReportingId` available to reporting.<br>
-&nbsp;&nbsp;Otherwise, interest group `name` available to reporting.<br>
-
-Here's a table representation of the above logic: (selectableBuyerAndSellerReportingId abreviated to selectableBASRI, buyerAndSellerReportingId abreviated to BASRI, buyerReportingId abreviated to BRI)
+Protected Audience provides several interest group fields that can be used to report details about a bid.  These fields needs to
+be jointly k-anonymous with the interest group owner, bidding script URL, and render URL to be provided to the reporting fuctions.
+Which of `selectableBuyerAndSellerReportingId`, `buyerAndSellerReportingId`, `buyerReportingId`, and the interest group name gets
+passed to `reportWin()` and `reportResult()` is determined by the browser with the following logic:
+
+If `selectableBuyerAndSellerReportingId` defined in interest group:
+* If `selectableBuyerAndSellerReportingId` in bid:
+    * Then `selectableBuyerAndSellerReportingId`, `buyerAndSellerReportingId` (if present in interest group), and `buyerReportingId`
+      (if present in interest group) will all be available to reporting.
+* Otherwise (i.e. when If `selectableBuyerAndSellerReportingId` not in bid):
+    * No reporting IDs. If you want reporting IDs, consider including and selecting an empty `selectableBuyerAndSellerReportingId`.
```

Comment:
To confirm, the 'empty' suggestion is to include 
   `selectableBuyerAndSellerReportingIds: [""]`
in all renderURLs and submit
   `selectedBuyerAndSellerReportingId: ""`
with bids?

Reply:
I added an example of an empty `selectableBuyerAndSellerReportingIds`.  Somewhere along the way I also added a blurb for `selectedbuyerAndSellerReportingIdRequired` and mentioned in two places how a seller might use it in `scoreAd`.

Reply:
Yes, thank you for the clarifications, &c. @JensenPaul ! 

I saw that example a few minutes ago and was going to confirm that the _send-an-empty-selected-value_ pattern/trick will unlock BASRI and BRI reporting irrespective of selectedBuyerAndSellerReportingIdRequired. Sellers should ignore these empty values (not consider them invalid) and score the bid normally.

Other things I noted as I scanned the last patch:

* typo: _abreviated_ (2x above the table)
* In the reporting side of the table should it indicate that reports get select**ed**BASRI.
* typo: _interst_ (below the table)
* The sample code below the table uses ‘ instead of '

I'm not clear on the take-away from the 
>If the same seat ID is always presented for bids on an ad...
>If different seat IDs are presented for bids on an ad: 

example.